bouncer.go

```diff
@@ -585,13 +630,23 @@ func crowdsecQuery(bouncer *Bouncer, stringURL string, isPost bool) ([]byte, err
 		if errToken := getToken(bouncer); errToken != nil {
 			return nil, fmt.Errorf("crowdsecQuery:renewToken url:%s %w", stringURL, errToken)
 		}
-		return crowdsecQuery(bouncer, stringURL, false)
+		return crowdsecQuery(bouncer, stringURL, false, nil, expectedStatusCodes)
 	}
-	if res.StatusCode != http.StatusOK {
-		return nil, fmt.Errorf("crowdsecQuery url:%s, statusCode:%d", stringURL, res.StatusCode)
+
+	// Check if the status code is in the expected list
+	isExpectedStatus := false
+	for _, expectedCode := range expectedStatusCodes {
+		if res.StatusCode == expectedCode {
+			isExpectedStatus = true
+			break
+		}
+	}
+
+	if !isExpectedStatus {
+		return nil, fmt.Errorf("crowdsecQuery method:%s url:%s, statusCode:%d (expected: %v)", req.Method, stringURL, res.StatusCode, expectedStatusCodes)
```

Comment:
I don't see the value of `expectedStatusCodes`, just test if it's in HTTP status 200 range (2XX)

Reply:
Done